storage-proofs-porep/src/stacked/vanilla/create_label/multi.rs

```diff
@@ -754,6 +754,9 @@ mod tests {
             .read_at(final_labels.len() - 1)
             .expect("read_at");
         dbg!(&last_label);
-        assert_eq!(expected_last_label.to_repr(), last_label.0);
+        assert_eq!(
+            expected_last_label.to_repr(),
+            <<Fr as PrimeField>::Repr>::from(last_label)
```

Comment:
It's just an aesthetic preference, but these should also work using the current `Domain` trait.
```rust
assert_eq!(expected_last_label.to_repr(), last_label.as_ref())
assert_eq!(expected_last_label.to_repr(), last_label.into_bytes()
```